### ISSUE TITLE
Service Facebook

### ISSUE THREAD

**Issue by DJTommek** [DJTommek/better-location#55]
## Service home website
https://www.facebook.com

## Example link(s) and their coordinates

Example link | Coordinates |  
-|-
https://cs-cz.facebook.com/burgerzelva/ | `50.08724438,14.46923018`
https://www.facebook.com/burgerzelva/ | `50.08724438,14.46923018`

## Example location link
N/A

## Example drive link
N/A

## Description (optional)
Coordinates might be gathered by:
- HERE link in "Get Directions" ([example](http://share.here.com/r/mylocation/e-eyJuYW1lIjoiXHUwMTdkZWx2YSBiZWVycyAmIGJ1cmdlcnMiLCJhZGRyZXNzIjoiSmFuYSBcdTAxN2RlbGl2c2tcdTAwZTlobyAxODA2XC8yMywgUHJhZ3VlLCBDemVjaCBSZXB1YmxpYyIsImxhdGl0dWRlIjo1MC4wODcyNDQzNzUwODMsImxvbmdpdHVkZSI6MTQuNDY5MjMwMTc1MDE4LCJwcm92aWRlck5hbWUiOiJmYWNlYm9vayIsInByb3ZpZGVySWQiOjU0MzA1OTg1MjUxNzE1MH0=?link=addresses&fb_locale=en_US&ref=facebook))
- static map image in "About" section ([example](https://external-prg1-1.xx.fbcdn.net/static_map.php?v=2009&osm_provider=2&size=306x98&zoom=15&markers=50.08724438%2C14.46923018&language=cs_CZ))

**Comment by DJTommek** [comment]
Scraping via CURL is working good on local, but not on production server - facebook is redirecting to login page.
Probably it has to be done using official Facebook API. 

tldr: I need to publish this app (BetterLocation) to review from Facebook staff to approve usage for public pages. Then I will be able to use permission to request public pages, like this: https://developers.facebook.com/tools/explorer/?method=GET&path=burgerzelva%3Ffields%3Did%2Cname_format&version=v9.0

Links to study:
- https://developers.facebook.com/tools/explorer/
- https://developers.facebook.com/docs/places/web
- https://developers.facebook.com/tools/accesstoken/
